client/elements/menus/sc-language-base-menu.js

```diff
@@ -85,15 +78,17 @@ class LanguageBaseMenu extends LitLocalized(LitElement) {
     }
 
     try {
-      this.selectedLanguageNum = this._findChosenLanguageIndex(event.detail.item.id);
+      this.selectedLanguageNum = this._findChosenLanguageIndex(event.target.id);
       const chosenLanguage = this.languageListResponse[this.selectedLanguageNum];
       // If it's not a main language change menu (but a clone), dispatch an event
       if (this.cloneName) {
-        dispatchEvent(new CustomEvent(`${this.cloneName}-language-changed`, {
-          detail: { isoCode: chosenLanguage.iso_code, name: chosenLanguage.name },
-          composed: true,
-          bubbles: true
-        }));
+        dispatchEvent(
```

Comment:
I know it's just formatting here, but I'm pretty sure error throws here, shouldn't it be `this.dispatchEvent`?

Reply:
I'm not able to check whether it's work, cloneName is not set anywhere. Looks like legacy code, I think we can remove this dispatch at all. @ihongda  Do you know anything about this event? Is it still used?